NEWS

```diff
@@ -131,6 +131,10 @@ Changed Functionality
   This prevents callbacks into script-land through change handlers when parts
   of the environment have already been torn down.
 
+- When running in cluster mode, the manager by default now imports metrics
+  from all other cluster nodes via broker topics and opens port 9911/tcp for
```

Comment:
I've been catching up on the telemetry work and I find it'd be helpful to explain somewhere with examples how Broker topics and their naming / structure relate to metrics collection, most likely over in `zeek-docs`.

Reply:
Agree on updating the docs to point out that metrics importing/centralization is now happening by default (the docs currently have some rambling around setting environment variables to enable this) and possibly how to turn it off.

As of how broker topics and naming relate to metrics collection, not sure it's `zeek-docs` material: It's an implementation detail that may or may not change down the road, too.

Reply:
Good point about the implementation side of things, that wasn't clear to me. Developer docs are a bit tricky — there is https://docs.zeek.org/en/master/devel/index.html, but we've also been using the wiki for such things. Please feel free to handle as you see fit.